gulpfile.js

```diff
@@ -312,8 +320,82 @@ function buildExperimentsJSON(path) {
   return through.obj(collectEntry, endStream);
 }
 
+gulp.task('stage-assets', function() {
+  return gulp.src(DEST_PATH + '**')
+      .pipe(gulp.dest(STAGE_PATH + 'static'));
+});
+
+gulp.task('move-api', function() {
+  return new Promise((resolve, reject) => {
+    gulp.src(STAGE_PATH + 'static/api/**', { base: STAGE_PATH + 'static' })
+    .pipe(gulp.dest(STAGE_PATH))
+    .on('end', () => {
+      del(STAGE_PATH + 'static/api').then(resolve).catch(reject);
+    });
+  });
+});
+
+gulp.task('copy-html', function() {
+  const paths = fs.readdirSync(CONTENT_SRC_PATH + 'experiments')
+    .map(f => `${STAGE_PATH}experiments/${f.replace('.yaml', '')}`)
+    .concat([
+      STAGE_PATH,
+      STAGE_PATH + 'experiments',
+      STAGE_PATH + 'onboarding',
+      STAGE_PATH + 'home',
+      STAGE_PATH + 'share',
+      STAGE_PATH + 'legacy',
+      STAGE_PATH + 'error'
+    ]);
+  return merge(
+    gulp.src(SRC_PATH + 'index.html')
+      .pipe(multiDest(paths)),
+    gulp.src('./legal-copy/privacy-notice.html')
+      .pipe(rename('index.html'))
+      .pipe(gulp.dest(STAGE_PATH + '/privacy')),
+    gulp.src('./legal-copy/terms-of-use.html')
+      .pipe(rename('index.html'))
+      .pipe(gulp.dest(STAGE_PATH + '/terms'))
+  );
```

Comment:
This is kind of a bummer. I wonder if there's a way we can do some sort of discovery to avoid having to [define everything twice](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/lib/router.js#L10-L21).